crates/plugin_lodash/src/lib.rs

```diff
@@ -26,6 +27,15 @@ use std::{ops::Deref, path::PathBuf, sync::Arc};
 mod error;
 mod mappings;
 
+/// Cache for reducing lots of IO operations(Caused by `nodejs::resolve()`)
+///
+/// This is BAD in some situations
+/// If you `transform` in multi-thread, and while thread1 is still transforming,
+/// thread2 may change this mappings.
+static CACHED: Lazy<RwLock<CacheItem>> =
```

Comment:
Using a tuple for more than two items may be hard to be maintained. Not to mention the third one is undocumented.

Reply:
Global `RwLock` is not performant, maybe choose something with a bucket to split this. E.g `dashmap`